src/cascade/input_data/emr.py

```diff
@@ -83,12 +84,14 @@ def _make_interpolators(csmr):
     """
     mean = {}
     stderr = {}
-    mean["both"] = LSQBivariateSpline(
-        csmr.age, csmr.time, csmr["mean"], sorted(csmr.age.unique()), sorted(csmr.time.unique()), kx=1, ky=1
-    )
-    stderr["both"] = LSQBivariateSpline(
-        csmr.age, csmr.time, csmr.standard_error, sorted(csmr.age.unique()), sorted(csmr.time.unique()), kx=1, ky=1
-    )
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", module="scipy.interpolate")
```

Comment:
Is this always safe? We don't need to filter on some more granular value?

Reply:
Changed to be very specific.

Reply:
Cool